src/writers/writer.ts

```diff
@@ -74,13 +75,33 @@ export class Writer {
 
   public write(text: string, width: number, height: number, options: IWriteOptions) {
     if (Writer.SupportedRotation.indexOf(options.textRotation) === -1) {
-      throw new Error("unsupported rotation - " + options.textRotation);
+      throw new Error("unsupported rotation - " + options.textRotation +
+        ". Supported rotations are " + Writer.SupportedRotation.join(", "));
+    }
+    if (options.textShear != null && options.textShear < -80 || options.textShear > 80) {
```

Comment:
Well 90 would be so narrow that there simply no room for text. The height of the sheared line would be 0. 80 was a ballpark, but it should be something < 90.